test/NuGet.Core.Tests/NuGet.Frameworks.Test/CompatibilityListProviderTests.cs

```diff
@@ -232,5 +232,48 @@ public void CompatibilityListProvider_NetStandard20Supporting()
             // count
             Assert.Equal(19, actual.Length);
         }
+        [Fact]
+        public void CompatibilityListProvider_NetStandard21Supporting()
+        {
+            // Arrange
+            var provider = CompatibilityListProvider.Default;
+
+            // Act
+            var actual = provider
+                .GetFrameworksSupporting(FrameworkConstants.CommonFrameworks.NetStandard21)
+                .Select(f => f.ToString())
+                .ToArray();
+
+            // Assert
+            // positive
+            Assert.Contains(".NETCoreApp,Version=v3.0", actual);
+            Assert.Contains(".NETStandard,Version=v2.1", actual);
+            Assert.Contains(".NETStandardApp,Version=v2.1", actual);
```

Comment:
I'm pretty sure netstandardapp 2.1 is deprecated.

Reply:
I have a question,  are we going to remove netstandardapp2.1 from the supporting list, or we just need to remove the assert line for netstandardapp2.1? Thanks!

Reply:
Did you add netstandardapp2.1 in the mapping? 

Does that test pass? 

See netstandard2.0 and see that netstandardapp2.0 is not tested.